data/mods/Magiclysm/mutations/mutations.json

```diff
@@ -8,5 +8,596 @@
+      "strength_damage": { "damage_type": "bash", "amount": 0.5 }
+    },
+    "spells_learned": [ [ "dragon_shell_black", 1 ] ],
+    "prereqs2": [ "DRAGON_SPELLS" ],
+    "anger_relations": [ [ "HUMAN", 5 ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "SMELLY2",
+    "copy-from": "SMELLY2",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "DEFORMED2",
+    "copy-from": "DEFORMED2",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "FANGS",
+    "copy-from": "FANGS",
+    "extend": {
+      "category": [ "DRAGON_BLACK" ],
+      "attacks": [
+        {
+          "attack_text_u": "You sink your fangs into %s",
+          "attack_text_npc": "%1$s sinks their fangs into %2$s",
+          "required_mutations": [ "DRAGON_MAW_BLACK" ],
+          "body_part": "mouth",
+          "chance": 20,
+          "base_damage": { "damage_type": "stab", "amount": 25 },
+          "strength_damage": { "damage_type": "stab", "amount": 0.25 }
+        }
+      ]
+    }
+  },
+  {
+    "type": "mutation",
+    "id": "MUZZLE_LONG",
+    "copy-from": "MUZZLE_LONG",
+    "extend": { "category": [ "DRAGON_BLACK" ] },
+    "changes_to": [ "DRAGON_MAW_BLACK" ]
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGON_MAW_BLACK",
+    "name": { "str": "Black Dragon Maw" },
+    "points": -3,
+    "visibility": 8,
+    "ugliness": 8,
+    "mixed_effect": true,
+    "description": "Your face and jaws have extended to a long, toothy dragon maw.  It's fearsome and causes grievous wounds, but prevents wearing mouth gear.",
+    "prereqs": [ "MUZZLE_LONG" ],
+    "category": [ "DRAGON_BLACK" ],
+    "threshreq": [ "THRESH_DRAGON_BLACK" ],
+    "social_modifiers": { "intimidate": 30 },
+    "types": [ "MUZZLE" ],
+    "restricts_gear": [ "mouth" ],
+    "attacks": {
+      "attack_text_u": "You bite a chunk out of %s",
+      "attack_text_npc": "%1$s bites a chunk out of %2$s",
+      "blocker_mutations": [ "FANGS" ],
+      "body_part": "mouth",
+      "chance": 20,
+      "base_damage": { "damage_type": "stab", "amount": 20 },
+      "strength_damage": { "damage_type": "stab", "amount": 0.25 }
+    },
+    "spells_learned": [ [ "acid_bite", 1 ] ],
+    "prereqs2": [ "DRAGON_MAGIC" ]
+  },
+  {
+    "type": "mutation",
+    "id": "VOMITOUS",
+    "copy-from": "VOMITOUS",
+    "extend": { "category": [ "DRAGON_BLACK" ] },
+    "leads_to": [ "DRAGON_BREATH_BLACK", "OVERLY_ACID_GUT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "OVERLY_ACID_GUT",
+    "name": { "str": "Overly Acidic Digestion" },
+    "description": "Your digestive issues have developed to become overly acidic which greatly impacts your ability to normally digest food, resulting in getting far less nutrition from the food you eat and making your need to eat more often.",
+    "points": -3,
+    "prereqs": [ "VOMITOUS" ],
+    "category": [ "DRAGON_BLACK" ],
+    "metabolism_modifier": 0.5,
```

Comment:
i think this might be backwards

Reply:
You're right. I'll modify nauseous as a leads to for this and have it as a prereq2 for the breath.